src/Microsoft.Framework.Logging.EventLog/project.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+    "version": "1.0.0-*",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/aspnet/logging"
+    },
+    "description": "Logger for Windows Event Log.",
+    "dependencies": {
+        "Microsoft.Framework.Logging.Abstractions": "1.0.0-*",
+        "Microsoft.Framework.NotNullAttribute.Sources": "1.0.0-*"
```

Comment:
Nice. Has the syntax been simplified? (IIRC, the previous syntax was `"Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }`)